helm/prometheus-rules/templates/alerting-rules/kvm.management-cluster.rules.yml

```diff
@@ -12,6 +12,28 @@ spec:
   groups:
   - name: kvm
     rules:
+    - alert: BlackboxExporterEndpointDown
+      annotations:
+        description: '{{`Blackbox exporter endpoint ({{ $labels.instance }}) is down.`}}'
+        opsrecipe: blackbox-exporter-endpoint-down/
+      expr: up{job="management-cluster-blackbox-ingress-http"} == 0
+      for: 15m
+      labels:
+        area: kaas
+        severity: page
+        team: rocket
+        topic: observability
+    - alert: BlackboxExporterEndpointNotResponding
```

Comment:
Added a new alert that actually uses the metrics reported by the blackbox exporter.